packages/manager/src/features/CloudPulse/Alerts/CreateAlert/NotificationChannel/AddNotificationChannel.tsx

```diff
@@ -0,0 +1,179 @@
+            backgroundColor:
+              theme.name === 'light'
+                ? theme.tokens.color.Neutrals[5]
+                : theme.tokens.color.Neutrals.Black,
+            borderRadius: 1,
+            p: 2,
+          })}
+        >
+          <Typography
+            gutterBottom
+            sx={(theme) => ({ color: theme.color.black })}
+            variant="body2"
+          >
+            Channel Settings
+          </Typography>
+          <Controller
+            render={({ field, fieldState }) => (
+              <Autocomplete
+                onChange={(
+                  _,
+                  newValue: { label: string; value: ChannelTypes },
+                  reason
+                ) => {
+                  if (reason === 'selectOption') {
+                    field.onChange(newValue?.value);
+                  }
+
+                  if (reason === 'clear') {
+                    field.onChange(null);
+                  }
+                }}
+                value={
+                  field.value !== null
+                    ? ChannelTypeOptions.find(
+                        (option) => option.value === field.value
+                      )
+                    : null
+                }
+                data-testid="channel-type"
+                errorText={fieldState.error?.message}
+                label="Type"
+                onBlur={field.onBlur}
+                options={ChannelTypeOptions}
+              />
+            )}
+            control={control}
+            name="channel_type"
+          />
+          <Box>
+            <Controller
+              render={({ field, fieldState }) => (
+                <Autocomplete
+                  onChange={(_, selected, reason) => {
+                    if (reason === 'selectOption') {
+                      field.onChange(selected?.value);
+                    }
+
+                    if (reason === 'clear') {
+                      field.onChange(null);
+                    }
+                  }}
+                  value={
+                    field.value !== null
+                      ? templateOptions.find(
+                          (option) => option.value === field.value
+                        )
+                      : null
+                  }
+                  data-testid="channel-label"
+                  disabled={selectedTypeTemplate === null}
+                  errorText={fieldState.error?.message}
+                  key={channelTypeWatcher}
+                  label="Channel"
+                  onBlur={field.onBlur}
+                  options={templateOptions}
+                />
+              )}
+              control={control}
+              name="label"
+            />
+          </Box>
+
+          {selectedTemplate && (
+            <Box paddingTop={2}>
+              <Grid container>
+                <Grid item md={2}>
+                  <Typography variant="h3">To:</Typography>
+                </Grid>
+                <Grid item md={10}>
```

Comment:
some extra conditions should be there to check if the type of template is email , i think